web/src/app/alerts/CreateAlertDialog/StepContent/Step1.js

```diff
@@ -0,0 +1,187 @@
+import React, { useRef } from 'react'
+import {
+  Grid,
+  TextField,
+  InputAdornment,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  Paper,
+  Typography,
+  InputLabel,
+  Chip,
+} from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { FormField } from '../../../forms'
+import ServiceLabelFilterContainer from '../../../services/ServiceLabelFilterContainer'
+import { Search as SearchIcon } from '@material-ui/icons'
+import FavoriteIcon from '@material-ui/icons/Star'
+import { ServiceChip } from '../../../util/Chips'
+import AddIcon from '@material-ui/icons/Add'
+
+const useStyles = makeStyles(theme => ({
+  addAll: {
+    backgroundColor: theme.palette.primary['400'],
+  },
+  chipContainer: {
+    padding: theme.spacing(0.5),
+    marginBottom: theme.spacing(2),
+    height: '9em',
+    overflow: 'auto',
+    border: '1px solid #bdbdbd',
+  },
+  endAdornment: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  noticeText: {
+    width: '100%',
+    textAlign: 'center',
+    alignSelf: 'center',
+    lineHeight: '9em',
+  },
+}))
+
+const CREATE_ALERT_LIMIT = 35
+
+export default props => {
+  const { formFields, queriedServices } = props
+
+  const fieldRef = useRef()
+  const classes = useStyles()
+
+  const labelKey = formFields.searchQuery.split(/(!=|=)/)[0]
+  const labelValue = formFields.searchQuery
+    .split(/(!=|=)/)
```

Comment:
This should be `.split('=')` since we only care about values AFTER the first `=`.

Otherwise, it will break with something like `label/key=value!=with=stuff`

Reply:
The current logic in `serviceRouter` is:

```
const searchSplit = searchParam.split(/(!=|=)/)
labelKey = searchSplit[0]
// the value can contain "=", so joining the rest of the match such that it doesn't get lost
labelValue = searchSplit.slice(2).join('')
```

it got updated during its PR

Reply:
We can probably make it a util function `getServiceLabel` that both use that returns
```
{ key: x, value: y }
```
so the logic remains the same in all locations